### PR TITLE
[shopsys] Migration from IvoryCKEditorBundle to FOSCKEditorBundle

### PR DESCRIPTION
**Pull request by boris-brtan**
- ckEditor package was replaced in composer.json
- FOSCKEditorBundle was registered via AppKernel
- ivory_ck_editor config was updated into fos_ck_editor config
- uses of namespaces for old ckEditor were updated with the new namespaces
- due to license reasons FOSCKEditor downloads frontend scripts via command ckeditor:install
- version of frontend part was locked into 4.5.11 to match previous skin and also the skin of fm_elfinder
- license of ckEditor frontent was introduced into open-source-license-acknowledgements-and-third-party-copyrights.md

| Q             | A
| ------------- | ---
|Description, reason for the PR| since we are moving to symfony ^4.0, 2 years stale project https://packagist.org/packages/egeloen/ckeditor-bundle doesn't support the version, however FOS group forked this project and make it supported also for symfony ^4.0 thus we migrated this package to https://packagist.org/packages/friendsofsymfony/ckeditor-bundle
|New feature| Yes 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| Yes 
|Fixes issues|  
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/upgrade/UPGRADE-v8.0.0.md [hunk 1]
```diff
@@ -28,6 +28,48 @@ There you can find links to upgrade notes for other versions too.
 
     In case of extending one of these classes, you should add an `EntityNameResolver` to a constructor and use it in a `create()` method to resolve correct class to return.
 - run `php phing standards-fix` so all nullable values will be now defined using nullability (?) symbol ([#1010](https://github.com/shopsys/shopsys/pull/1010))
+- change `IvoryCKEditorBundle` into `FOSCKEditorBundle` ([#1072](https://github.com/shopsys/shopsys/pull/1072))
```
Comment (vitek-rostislav) [inline docs/upgrade/UPGRADE-v8.0.0.md]:
```suggestion
- replace `IvoryCKEditorBundle` with `FOSCKEditorBundle` ([#1072](https://github.com/shopsys/shopsys/pull/1072))
```

File: docs/upgrade/UPGRADE-v8.0.0.md [hunk 2]
```diff
@@ -28,6 +28,48 @@ There you can find links to upgrade notes for other versions too.
...
+    - replace the registration instance of the bundle in `app/AppKernel`
```
Comment (vitek-rostislav) [inline docs/upgrade/UPGRADE-v8.0.0.md]:
```suggestion
    - replace the registration of the bundle in `app/AppKernel`
```

File: docs/upgrade/UPGRADE-v8.0.0.md [hunk 3]
```diff
@@ -28,6 +28,48 @@ There you can find links to upgrade notes for other versions too.
...
+        ```diff
+        - new Ivory\CKEditorBundle\IvoryCKEditorBundle(), // has to be loaded after FrameworkBundle and TwigBundle
+        + new FOS\CKEditorBundle\FOSCKEditorBundle(), // has to be loaded after FrameworkBundle and TwigBundle
+        ```
+    - replace the name of the `app/config/packages/ivory_ck_editor.yml` into `app/config/packages/fos_ck_editor.yml` and its content
```
Comment (vitek-rostislav) [inline docs/upgrade/UPGRADE-v8.0.0.md]:
```suggestion
    - rename `app/config/packages/ivory_ck_editor.yml` to `app/config/packages/fos_ck_editor.yml` and change the root key in its content
```

File: docs/upgrade/UPGRADE-v8.0.0.md [hunk 4]
```diff
@@ -28,6 +28,48 @@ There you can find links to upgrade notes for other versions too.
...
+        ```diff
+        - ivory_ck_editor:
+        + fos_ck_editor:
+        ```
+    - change the package in `composer.json`
+        ```diff
+        - "egeloen/ckeditor-bundle": "^4.0.6",
+        + "friendsofsymfony/ckeditor-bundle": "^2.1",
+        ```
+    - update all usages of old bundle in
```
Comment (vitek-rostislav) [inline docs/upgrade/UPGRADE-v8.0.0.md]:
```suggestion
    - update all usages of the old bundle in
```

File: packages/framework/src/Command/ComposerScriptHandler.php [hunk 5]
```diff
@@ -17,6 +17,7 @@ public static function postInstall(Event $event)
     {
         static::executeCommands($event, [
             'shopsys:domains-urls:configure',
+            'ckeditor:install --clear=skip --release=full --tag=4.5.11',
```
Comment (vitek-rostislav) [inline packages/framework/src/Command/ComposerScriptHandler.php]: I believe this is covered by the two points in commit message
```
- due to license reasons FOSCKEditor downloads frontend scripts via command ckeditor:install
- version of frontend part was locked into 4.5.11 to match previous skin and also the skin of fm_elfinder
```

However, I would appreciate a little bit more information. Why it is not implemented as a phing target? Does it need to be locked? Why do we wanna keep the previous skins? :grin: 
Maybe it would make sense to extract this change into a separate commit? :thinking:

Comment (boris-brtan) [reply packages/framework/src/Command/ComposerScriptHandler.php]: splited into 2 commits with link onto symfony documentation about this package.

**Comment by boris-brtan** [comment]
rebased, problem with phpstan, phpunit that will be merged from master/7.2 branch
